onnx/backend/test/cpp/driver_test.cc

```diff
@@ -160,6 +163,10 @@ class ONNXCppDriverTest
       }
       std::string serialized_model;
       model_.SerializeToString(&serialized_model);
+      ASSERT_EQ(
+          lib.onnxGetBackendCompatibility(
+              backendID, serialized_model.size(), serialized_model.data()),
+          ONNXIFI_STATUS_SUCCESS);
```

Comment:
So if it is not compatible, do we need to record/report something?

Reply:
Right now we will fail this test directly. Guess if we should skip this test once we found it is not compatible?

Reply:
I think it's a valid response if the model cannot be supported by the ONNXIFI implementation.

I'd be in favor to make the test pass if the response is `ONNXIFI_STATUS_UNSUPPORTED_OPERATOR` or `ONNXIFI_STATUS_SUCCESS`.

* In case of `ONNXIFI_STATUS_SUCCESS` do other checks as we do currently.
* In case of `ONNXIFI_STATUS_UNSUPPORTED_OPERATOR` make test pass and report warning for the given test. 
* In case of any other results fail the test immediately.

Reply:
@houseroad @rdzhabarov Just realized gtest doesnt have skipping... I will let these test passed. 
https://github.com/google/googletest/pull/1544